change/@fluentui-web-components-724db53b-8216-4544-99e5-0c2d440e4817.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28717&commitId=bac89b66d0cddfa3b5c0194a559f3cf547013698&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Disabled - RTL.normal.chromium.png|3| Changed|
|Accordion.Expand Icon Position End.normal.chromium.png|5| Changed|
|Accordion.Disabled - Dark Mode.normal.chromium.png|19| Changed|
|Accordion.Expand Icon Position End - Dark Mode.normal.chromium.png|33| Changed|
|Accordion.Expand Icon Position End - RTL.normal.chromium.png|5| Changed|
|Accordion.Size - Dark Mode.normal.chromium.png|66| Changed|
|Accordion.Disabled.normal.chromium.png|3| Changed|
|Accordion.Size.normal.chromium.png|11| Changed|
|Accordion.Size - RTL.normal.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>